src/main/drivers/system.c

```diff
@@ -273,6 +273,18 @@ void initialiseMemorySections(void)
 #endif
 }
 
+void initialiseD2MemorySections(void)
```

Comment:
This seems to be only used for STM32H7 - should this be conditional?

Reply:
Whoops. It was already in system.h. Fixed.